brave/brave-core
# Deprecate Brave Ads DoesAdTypeSupportFrequencyCapping

Resolves https://github.com/brave/brave-browser/issues/20983

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm conversion frequency cap is working as expected
- Confirm daily frequency cap (campaign) is working as expected
- Confirm per day frequency cap (creative set) is working as expected
- Confirm per hour frequency cap (creative_instance_id) is working as expected
- Confirm per month frequency cap (creative set) is working as expected
- Confirm per week frequency cap (creative set) is working as expected
- Confirm total max frequency cap (creative set) is working as expected
- Confirm transferred frequency cap (campaign) is working as expected

^^^ for ad notifications, new tab page ads and inline content ads (also test different ad types within the same campaign/creative which should have their own caps.
